### PR TITLE
tests squiggly heredoc

### PR DESCRIPTION
**Pull request by sgarciac**
* See #175
* See [Feature 9098]

### CODE REVIEW

**Comment by sgarciac** [comment]
ouch, it breaks on rubies < 2.3, obviousl. I'll come back with a new PR.

**Comment by sgarciac** [comment]
I'm using eval now, to prevent syntax problems on rubies < 2.3

File: language/string_spec.rb [hunk 1]
```diff
@@ -176,6 +176,13 @@
     s.should == '    foo bar#{@ip}' + "\n"
   end
 
+  ruby_version_is "2.3" do
+    it "allows HEREDOC with <<~'identifier', allowing to indent identifier and content" do
```
Comment (eregon) [inline language/string_spec.rb]: I see, you meant the heredoc identifier, not the variable.
I think it's worth moving this to a fixture and under a method (say `StringSpec.squiggly`) to improve readability and make it a more real-world example.
You can then `require fixture(__FILE__, "squiggly_heredoc.rb")`

Comment (sgarciac) [inline language/string_spec.rb]: ok

**Comment by sgarciac** [comment]
new version

**Comment by eregon** [comment]
Well done, thanks!

**Pull request merged**